packages/nextjs/scaffold.config.ts

```diff
@@ -14,6 +14,7 @@ export const DEFAULT_ALCHEMY_API_KEY = "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF";
 const scaffoldConfig = {
   // The networks on which your DApp is live
   targetNetworks: [chains.arbitrum],
+  // targetNetworks: [chains.hardhat],
```

Comment:
It's bad practice to leave commented code. Other contributors might be confused about whether it should be uncommented or removed.